codespell_lib/data/dictionary.txt

```diff
@@ -16654,9 +16722,11 @@ midified->modified
 migrateable->migratable
 migt->might, midget,
 migth->might
+miinimum->minimum
```

Comment:
Can you add the -ise, -ize, -ised, -ized, -ising, -izing, -isation and -ization versions of this typo too.